spec/serializers/rest/account_serializer_spec.rb

```diff
@@ -44,4 +45,44 @@
       expect(subject['memorial']).to be true
     end
   end
+
+  context 'when created_at is populated' do
+    before do
+      account.account_stat.update!(created_at: default_datetime)
+    end
+
+    it 'parses as RFC 3339 datetime' do
+      expect { DateTime.rfc3339(subject['created_at']) }.to_not raise_error
+    end
+  end
+
+  context 'when last_status_at is populated' do
+    before do
+      account.account_stat.update!(last_status_at: default_datetime)
+    end
+
+    it 'is serialized as yyyy-mm-dd' do
+      expect(subject['last_status_at']).to eq('2024-11-28')
+    end
+  end
+
+  context 'when mute_expires_at is populated', pending: 'creating a muted account' do
+    before do
+      account.account_stat.update!(mute_expires_at: default_datetime)
+    end
+
+    it 'parses as RFC 3339 datetime' do
+      expect { DateTime.rfc3339(subject['mute_expires_at']) }.to_not raise_error
+    end
+  end
+
+  context 'when verified_at is populated', pending: 'creating a verified account' do
```

Comment:
These "verified" come from the "fields" on account. You can get a field with something like....

`Account::Field.new(account, 'name' => 'Foo', 'value' => 'Bar', 'verified_at' => 3.days.ago)`

...and then would need `Account#fields` to return an array w/ that in it.

Reply:
Thanks, that was the pointer I needed.